spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -398,7 +398,7 @@ bom {
 			]
 		}
 	}
-	library("Hibernate", "5.4.23.Final") {
+	library("Hibernate", "5.4.24.Final") {
```

Comment:
The latest version should be 5.2.25.Final

Reply:
Thanks for the review. I have made the change.